plugins/utils.go

```diff
@@ -0,0 +1,26 @@
+package plugins
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+// LoopExecConfig is configuration used to register the LOOP and generate an exec
+type LoopExecConfig struct {
+	ID  string // unique string used by the node to track the LOOP. typically supplied by the loop logger name
+	Cmd string // string value of executable to exec
+	LoggingConfig
+}
+
+// MakeLoopCmd is helper to ensure synchronization between the loop registry and os cmd to exec the LOOP
+func MakeLoopCmd(loopRegistry *LoopRegistry, lcfg LoopExecConfig) (func() *exec.Cmd, error) {
```

Comment:
Should we call this something like `CmdFactory` or `CmdFn`? 
Why `Make` instead of `New`?
```suggestion
func NewCmdFactory(loopRegistry *LoopRegistry, lcfg LoopExecConfig) (func() *exec.Cmd, error) {
```